content/en/news/releases/1.23.x/announcing-1.23/change-notes/index.md

```diff
@@ -0,0 +1,155 @@
+
+- **Fixed** an issue that span name isn't set when using the OpenTelemetry tracing provider.
+
+- **Fixed** statsMatcher's regex which didn't match route's stat_prefix
+
+- **Fixed** an issue where the `cluster_name` and `http_conn_manager_prefix` label was incorrectly truncated for services without a `.svc.cluster.local` suffix
+
+- **Removed** Istio Stackdriver metrics from XDS.
+  ([Issue #50808](https://github.com/istio/istio/issues/50808))
+
+- **Removed** Istio Stackdriver logs from XDS.  ([Issue #50808](https://github.com/istio/istio/issues/50808))
+
+- **Removed** OpenCensus tracer from Istio XDS.  ([Issue #50808](https://github.com/istio/istio/issues/50808))
+
+- **Removed** the feature flag `ENABLE_OTEL_BUILTIN_RESOURCE_LABELS`.
+
+## Extensibility
+
+- **Removed** internal multi-version protobuf files from the API. This is an internal change for most users. If you directly consume Istio APIs as protobufs, read the upgrade notes. ([Issue #3127](https://github.com/istio/api/issues/3127))
+
+## Installation
+
+- **Improved** Added `.Values.pilot.trustedZtunnelNamespace` to `istiod` chart. Set this if installing ztunnel to a different namespace from `istiod`. Supercedes `.Values.pilot.env.CA_TRUSTED_NODE_ACCOUNTS` (which is still respected if set)
+
+- **Improved** the Helm installation for Istiod multi-cluster for primary-remote. Now, Helm installation only requires setting global.externalIstiod, instead of also requiring pilot.env.EXTERNAL_ISTIOD to be set.
+  ([Issue #51595](https://github.com/istio/istio/issues/51595))
+
+- **Updated** the [`distroless`](/docs/ops/configuration/security/harden-docker-images/) images to be based on [Wolfi](https://wolfi.dev).
+This should have no user-facing impact.
+
+- **Updated** Kiali addon to version v1.87.0.
+
+- **Added** the `releaseChannel:extended` flag to non-GA features and APIs. ([Issue #173](https://github.com/istio/enhancements/issues/173))
+
+- **Added** outlier log path configuration in mesh proxy config which allows users to configure the path to the outlier detection log file. ([Issue #50781](https://github.com/istio/istio/issues/50781))
+
+- **Added** Add an `ambient` umbrella chart that does nothing but wrap the baseline Istio components required for installing Istio with ambient support
+
+- **Added** Istiod's readiness check is now also available over https for use in clusters utilizing a remote control plane for sidecar injection. ([Issue #51506](https://github.com/istio/istio/issues/51506))
+
+- **Fixed** Ensure CNI plugin inherits CNI agent log level, simplify CNI logging config
+  ([Issue #50958](https://github.com/istio/istio/issues/50958))
+
+- **Fixed** service account annotation formatting error fixed by removing dashes ([Issue #51289](https://github.com/istio/istio/issues/51289))
+
+- **Fixed** `istio.io/rev` is not user intent for ztunnel, should be an annotation. Also, consistently propagate custom annotations in the ztunnel chart
```

Comment:
```suggestion
- **Fixed** an issue where custom annotations were not propagated to the ztunnel chart.
```

I don't understand the first part... This might need to be checked by finding the Issue and seeing what was meant there.